typescript/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@process-engine/consumer_api_contracts",
-  "version": "8.0.0",
+  "version": "8.0.1",
```

Minor, nicht patch 😉 

```suggestion
  "version": "8.1.0",
```

---

🤦‍♂